CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## 2.13.0-rc.0 / 2019-09-25
+
+* [CHANGE] Metrics: Change prometheus_sd_configs_failed_total to Gauge #5254
```

Comment:
Consider mentioning that the name is now `prometheus_sd_failed_configs`.

Reply:
updated